SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+                    Spacer()
+                }
+                
+            }
+           
+        }
+            .navigationBarBackButtonHidden()
+        
+        
+    }
+    
+    private var toolbarTitleTextView: some View {
+        Text("CLIMATE")
+    }
+    
+    private var backButtonView: some View {
+        СonvexCircleButtonView(actionHandler: {
+            presentation.wrappedValue.dismiss()
+        }, iconName: "chevron").tint(.white)
+    }
+    
+    private var settingsButtonView: some View {
+        СonvexCircleButtonView(actionHandler: {
+            print("")
+        }, iconName: "settings").tint(.white)
+    }
+        
+//    private var gradientView: GradientsBackgroundStackView {
+//
+//
+// GradientsBackgroundStackView(fromGradientColors: welcomeViewModel.lockGradientColors, toGradientColors: welcomeViewModel.unlockGradientColors, progress: welcomeViewModel.gradientProgress)
+//    }
+    
+    private var temperatureIndicatorView: some View {
+        return ZStack {
+            Circle()
+                .fill(LinearGradient(colors: [Color.black.opacity(0.3), Color.white.opacity(0.1)], startPoint: .top, endPoint: .bottomTrailing))
+                .frame(width: 140)
+                .overlay(Circle().stroke(LinearGradient(colors: [Color("RoundBack1"), Color("RoundBack2")], startPoint: .topLeading, endPoint: .bottomTrailing), lineWidth: 28))
+                .background(Circle().fill(Color("back2")))
+                .shadow(color: Color("ligth").opacity(0.1), radius: 25, x: -17, y: -15)
+                .shadow(color: Color("dark").opacity(0.5), radius: 20, x: 15, y: 15)
+            Circle()
+                .fill(.clear)
+                .frame(width: 110)
+            
+                .overlay(Circle().stroke(LinearGradient(colors: [Color(.black).opacity(0.2), Color(.white).opacity(0.1), Color(.black).opacity(0.2)], startPoint: .topLeading, endPoint: .bottomTrailing), lineWidth: 1.5))
+            
+            Circle()
+                .trim(from: 0, to: climateControlViewModel.isClimateControlEnabled ? climateControlViewModel.climateSliderValues[climateControlViewModel.settingsIndex] : 0)
+                .stroke(LinearGradient(colors: [Color(climateControlViewModel.bottomGradientColor), Color(climateControlViewModel.topGradientColor)], startPoint: .top, endPoint: .bottomTrailing), lineWidth: 15)
+                .frame(width: 168)
+                .rotationEffect(.degrees(-90))
+                .shadow(color: Color(climateControlViewModel.bottomGradientColor).opacity(0.5), radius: 3, x: -3, y: -3)
+                .shadow(color: Color(climateControlViewModel.topGradientColor).opacity(0.5), radius: 3, x: 3, y: 3)
+        }
+    }
+    
+    private var gradient: LinearGradient {
+        LinearGradient(colors: [Color(Constants.gradientTopColorName), Color(Constants.gradientBottomColorName)], startPoint: .bottom, endPoint: .top)
+    }
+    
+    private var bottomSheetView: some View {
```

Comment:
```suggestion
    private var bottomSheetView: some View {
```
в отдельное вью (и файл) вынеси